interview-questions.md

```diff
@@ -7,44 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer:
+A foreign key will be created in the Student Model since the association is one to many from Cohort to Students. The foreign key can be added by adding a column to the student model. The name of the column added would have the name of students_id.
```

Comment:
Great job adding migration in the researched answer!  In an interview, mentioning migration to add the foreign key will show understanding of the process even if you don't have the syntax available.